tox.ini

```diff
@@ -25,6 +26,7 @@ deps =
     django19: Django==1.9
     django110: Django==1.10
     django111: Django==1.11
+    djangodev: https://github.com/django/django/tarball/master
```

Comment:
I think you can use `django20: Django==2.0`now that it has been released to pypi :)